cmd/integration/integration.go

```diff
@@ -855,6 +879,29 @@ func ServeCachedManifestFile() (servingAddress string) {
 }
 
 const (
+	testManifestFile = `kind: Pod
```

Comment:
This change isn't exactly what we would like to do, because it only tests whether reading Pods works.
Do you have any suggestions how to make it test both scenarios (without significant changes to this file)?